tests/integration/bigquery.spec.ts

```diff
@@ -134,4 +143,15 @@ describe("@dataform/integration/bigquery", () => {
       });
     }
   });
+
+  describe("metadata", async () => {
+    it("includes jobReference and statistics", async () => {
+      const query = `select 1 as test`;
+      const { metadata } = await dbadapter.execute(query, { interactive: false, maxResults: 2 });
+      const { bigquery: bqMetadata } = metadata;
+      expect(bqMetadata).to.have.property("jobId");
```

Comment:
can we assert something a little stronger here? e.g. jobId matches some regexp, has >0 characters? and something about the bytes counts?

Reply:
Add regex match for uuid